beets/ui/commands.py

```diff
@@ -973,6 +973,8 @@ def list_func(lib, opts, args):
 
 
 list_cmd = ui.Subcommand('list', help='query the library', aliases=('ls',))
+list_cmd.parser.add_help_text('query the library')
+list_cmd.parser.add_help_text('Example usage: %prog -f \'$album: $title\' artist:foo')
```

Comment:
Also, it would be awesome to follow the "no contrived examples" rule and use a real artist name (or fragment of an artist name) instead of "foo". So, perhaps, `artist:beatles` or something.